docs/README.iOS.md

```diff
@@ -143,51 +143,66 @@ make -j$(getconf _NPROCESSORS_ONLN) -C tools/depends/target/binary-addons ADDONS
 Before you can use Xcode to build Kodi, the Xcode project has to be generated with CMake. CMake is built as part of the dependencies and doesn't have to be installed separately. A toolchain file is also generated and is used to configure CMake.
 
 ### 6.1. Build with Xcode
-Create an out-of-source build directory:
-```
-mkdir $HOME/kodi-build
-```
 
-Change to build directory:
+Change to Kodi's source code directory:
 ```
-cd $HOME/kodi-build
+cd $HOME/kodi
 ```
 
 Generate Xcode project for ARM 64bit (**recommended**):
 ```
-/Users/Shared/xbmc-depends/x86_64-darwin17.5.0-native/bin/cmake -G Xcode -DCMAKE_TOOLCHAIN_FILE=/Users/Shared/xbmc-depends/iphoneos11.3_arm64-target-debug/share/Toolchain.cmake ../kodi
+make -C tools/depends/target/cmakebuildsys GEN=Xcode
 ```
 
-Or generate Xcode project for ARM 32bit:
-```
-/Users/Shared/xbmc-depends/x86_64-darwin15.6.0-native/bin/cmake -G Xcode -DCMAKE_TOOLCHAIN_FILE=/Users/Shared/xbmc-depends/iphoneos9.3_armv7-target/share/Toolchain.cmake ../kodi
-```
+Additional cmake arguments can be supplied via the CMAKE_EXTRA_ARGUMENTS command line variable
+
+Example to generate Xcode project for ARM 32bit:
+````
+make -C tools/depends/target/cmakebuildsys GEN=Xcode CMAKE_EXTRA_ARGUMENTS="-DCMAKE_TOOLCHAIN_FILE=/Users/Shared/xbmc-depends/iphoneos9.3_armv7-target/share/Toolchain.cmake"
+````
 
-**WARNING:** The toolchain file location differs depending on your iOS and SDK version. You have to replace `x86_64-darwin15.6.0-native` and `iphoneos11.3_arm64-target-debug` or `iphoneos9.3_armv7-target` in the paths above with the correct ones on your system.
+**WARNING:** The toolchain file location differs depending on your iOS and SDK version. You have to replace `iphoneos11.3_arm64-target-debug` or `iphoneos9.3_armv7-target` in the paths above with the correct ones on your system.
```

Comment:
not needed as it is autodetected

Reply:
Left it in reference to the 32bit example for the cmake_extra_arguments and the 32bit toolchain file

Reply:
That isn't the correct way for building 32bit

Reply:
all it does is create the xcodeproj files, same as existing docs. Not sure i understand where the fault is? the produced project is exactly what would be produced by the current docs